src/genie/libs/parser/asa/tests/ShowInventory/cli/equal/golden_output_output.txt

```diff
@@ -1,5 +1,4 @@
-
-ciscoasa> show inventory
+ciscoasa# show inventory
```

Comment:
I don't know why this was altered but the whole line should actually be removed so that there is only device output.

Reply:
Agree on this one, line should be removed.